flask_monitoringdashboard/controllers/requests.py

```diff
@@ -30,6 +33,21 @@ def get_num_requests_data(db_session, start_date, end_date):
     }
 
 
+def get_status_code_distribution(db_session, endpoint_id):
+    results = db_session.query(Request.status_code, func.count(Request.status_code)).filter(
+        Request.endpoint_id == endpoint_id, Request.status_code.isnot(None)).group_by(Request.status_code).all()
+
+    total_count = 0
```

Comment:
```suggestion
    total_count = sum([f for (_, f) in results])
```

Reply:
Thanks, I like this suggestion. I removed the `[]` within the `sum()` and renamed `f` to `frequency`. I think `frequency` is a bit easier to read.